src/main/java/school/lemon/changerequest/java/generics/NumberGenerator/Printer.java

```diff
@@ -0,0 +1,34 @@
+package school.lemon.changerequest.java.generics.NumberGenerator;
+
+/**
+ * Created by lera on 27.12.16.
+ */
+public class Printer<T extends Number> {
+
+    NumberGeneratorImp numb;
```

Comment:
Please add generic parameter here. Thus you can avoid type casting in `generateAndPrint` method.